2016/05/what-is-react.js-and-why-use-react.md

```diff
@@ -0,0 +1,41 @@
+I have been using [React.js](https://facebook.github.io/react/index.html), a JavaScrip library for creating user interfaces, for about 4 months now.  You might want to read the official [getting-started](https://facebook.github.io/react/docs/getting-started.html) or [Tutorial](https://facebook.github.io/react/docs/tutorial.html) pages.
+
+**React has the following advantages:**
```

Comment:
Maybe something like. "From my point of view, React has............."  or something like that... "So far, I've discovered the following advantages"